cmd/fyne_demo/tutorials/bind.go

```diff
@@ -37,6 +37,31 @@ func bindingScreen(_ fyne.Window) fyne.CanvasObject {
 	checkLabel := widget.NewLabelWithData(binding.BoolToString(boolData))
 	checkEntry := widget.NewEntryWithData(binding.BoolToString(boolData))
 	checks := container.NewGridWithColumns(3, check, checkLabel, checkEntry)
+	item := container.NewVBox(floats, slide, bar, buttons, widget.NewSeparator(), checks, widget.NewSeparator())
 
-	return container.NewVBox(floats, slide, bar, buttons, widget.NewSeparator(), checks)
+	dataList := binding.NewFloatList()
+	dataList.Append(0.1)
+	dataList.Append(0.2)
+	dataList.Append(0.3)
+
+	button := widget.NewButton("Append", func() {
+		dataList.Append(float64(dataList.Length()+1) / 10)
+	})
+
+	list := container.NewBorder(nil, button, nil, nil, widget.NewListWithData(dataList,
+		func() fyne.CanvasObject {
+			return container.NewBorder(nil, nil, nil, widget.NewButton("+", nil),
+				widget.NewLabel("item x.y"))
+		},
+		func(item binding.DataItem, obj fyne.CanvasObject) {
+			text := obj.(*fyne.Container).Objects[0].(*widget.Label)
+			text.Bind(binding.FloatToStringWithFormat(item.(binding.Float), "item %0.1f"))
+
+			btn := obj.(*fyne.Container).Objects[1].(*widget.Button)
+			btn.OnTapped = func() {
+				item.(binding.Float).Set(item.(binding.Float).Get() + 1)
+			}
+		}))
+
+	return container.NewBorder(item, nil, nil, nil, list)
```

Comment:
Could these two border layouts be merged?
```suggestion
	list := widget.NewListWithData(dataList, ...
	return container.NewBorder(item, button, nil, nil, list)
```

Reply:
Yes, good catch thanks